CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Updates the `Button` Style Guide documentation.
 - Updates the `Iconography` Style Guide story documentation.
 - Updates the `Button` font weight to "regular".
+- Updates the organization of SCSS files by deleting some files and combining others.
```

Comment:
It looks like this doesn't break anything for consumers that were using the mixins before they can continue using mixins as they had?

Reply:
Yes, those mixins are still exported from the `resources.scss` file.